src/modules/shared-components/Navbar/index.jsx

```diff
@@ -25,24 +25,20 @@ export const Navbar = ({ isLoggedIn, onLogout }) => {
         <li>
```

Comment:
1. Remove `<li><Link to="/register"></Link></li>` above

2. `<li><Link to="/register">Register</Link></li>` should only be shown to users that **aren't logged in**
3. Create New Recipe should only be shown to users that **are logged in**